src/navigation.js

```diff
@@ -137,6 +147,10 @@ export default class TenUpNavigation {
 			$anchor.setAttribute( 'aria-controls', submenuID );
 			$anchor.setAttribute( 'aria-haspopup', true );
 
+			if ( 'click' === this.settings.action ) {
```

Comment:
Let's be liberal with the comments and pop something in the describe this addition